changelogs/unreleased/2896-datetime-encode-as-utc.yml

```diff
@@ -0,0 +1,7 @@
+description: Convert datetime objects to UTC prior to serialization
+change-type: patch
```

Comment:
This changes the behavior of the API. Shouldn't this be a `major`?

Reply:
I've thought about it and I don't know to be honest. If we make it a major we can only apply it to iso5.

Reply:
I don't think we should merge this into ISO4, do we?

Reply:
Well, we could consider that it more strictly defines the current behavior (now the timezone should be UTC, as the server is assumed to be on UTC, but it may or may not be UTC), so we close a hole in the specification